etc/kayobe/inventory/group_vars/all/firewall

```diff
@@ -0,0 +1,437 @@
+      network: "{{ provision_oc_net_name }}"
+      state: enabled
+    enabled: true
+
+stackhpc_ansible_control_infra_vm_firewalld_rules_extra: []
+
+###############################################################################
+# Seed firewalld rules
+
+stackhpc_seed_firewalld_rules: >-
+  {{ stackhpc_seed_firewalld_rules_default |
+    selectattr('enabled', 'true') |
+    map(attribute='rules') |
+    flatten |
+    selectattr('network', 'in', network_interfaces) |
+    selectattr('zone') |
+    union(stackhpc_seed_firewalld_rules_extra) |
+    unique |
+    select }}
+
+stackhpc_seed_firewalld_rules_default:
+  # Common
+  - rules:
+      - service: ssh
+        zone: "{{ admin_oc_net_name | net_zone }}"
+        network: "{{ admin_oc_net_name }}"
+        state: enabled
+      - service: dhcp
+        zone: "{{ provision_oc_net_name | net_zone }}"
+        network: "{{ provision_oc_net_name }}"
+        state: enabled
+      - service: tftp
+        zone: "{{ provision_oc_net_name | net_zone }}"
+        network: "{{ provision_oc_net_name }}"
+        state: enabled
+      - service: ntp
+        zone: "{{ provision_oc_net_name | net_zone }}"
+        network: "{{ provision_oc_net_name }}"
+        state: enabled
+      # Disable default services in public zone
+      - service: dhcpv6-client
+        state: disabled
+        zone: "{{ public_net_name | net_zone }}"
+        network: "{{ public_net_name }}"
+    enabled: true
+  - rules:
+      - service: ssh
+        state: disabled
+        zone: "{{ public_net_name | net_zone }}"
+        network: "{{ public_net_name }}"
+    enabled: "{{ public_net_name | net_zone != provision_oc_net_name | net_zone }}"
+  # Pulp server
+  - rules:
+      - service: http
+        zone: "{{ provision_oc_net_name | net_zone }}"
+        network: "{{ provision_oc_net_name }}"
+        state: enabled
+      # nginx
+      - port: 8080/tcp
+        zone: "{{ provision_oc_net_name | net_zone }}"
+        network: "{{ provision_oc_net_name }}"
+        state: enabled
+    enabled: "{{ seed_pulp_container_enabled | bool }}"
+  # Squid proxy
+  - rules:
+      - service: squid
+        zone: "{{ provision_oc_net_name | net_zone }}"
+        network: "{{ provision_oc_net_name }}"
+        state: enabled
+    enabled: "{{ seed_squid_container_enabled | bool }}"
+  # Ironic
+  - rules:
+      # Ironic inspector API
+      - port: 5050/tcp
+        zone: "{{ provision_oc_net_name | net_zone }}"
+        network: "{{ provision_oc_net_name }}"
+        state: enabled
+      # Ironic API
+      - port: 6385/tcp
+        zone: "{{ provision_oc_net_name | net_zone }}"
+        network: "{{ provision_oc_net_name }}"
+        state: enabled
+    enabled: true  #FIXME: Make rules conditional on Bifrost deployment
+  # Redfish exporter
+  - rules:
+      - port: 9610/tcp
+        zone: "{{ provision_oc_net_name | net_zone }}"
+        network: "{{ provision_oc_net_name }}"
+        state: enabled
+    enabled: true
```

Comment:
This one isn't in SKC yet, see https://github.com/stackhpc/stackhpc-kayobe-config/pull/501

Reply:
Probably will be once this patch gets merged though?

Reply:
It's now merged